services/alertmanager/alertmanager.go

```diff
@@ -78,7 +82,15 @@ func (svc *Service) Run(ctx context.Context) {
 // TODO That's a temporary measure until we start generating /etc/alertmanager.yml
 // using /srv/alertmanager/alertmanager.base.yml as a base. See supervisord config.
 func (svc *Service) generateBaseConfig() {
-	const path = "/srv/alertmanager/alertmanager.base.yml"
+	// create alert manager directories if not exists in the persistent volume.
+	if _, err := os.Stat(alertmanagerDataDir); os.IsNotExist(err) {
+		svc.l.Infof("Creating %q", alertmanagerDataDir)
+		if err := os.MkdirAll(alertmanagerDataDir, os.ModePerm); err != nil {
```

Comment:
That should be `0775` (rwxrwxr-x), not os.ModePerm. That's just not how it works – os.ModePerm is a bit mask, not a value.

We also should also set owner and group to the same values as `/srv/prometheus` has.

Reply:
it's strange, but with os.ModePerm it worked better :)

Reply:
os.ModePerm == 0777 == rwxrwxrwx. We should not allow everyone to write to that directory